### DISCUSSION TITLE
SecurityAccessDeniedException: Access is denied

### DISCUSSION THREAD

**Discussion by MichaelMatsiy** [dnnsoftware/Dnn.Platform#4904] [General] [answered]
Hi all,

I had been trying to setup a dnn instance on the new server but faced with the following error:

> **[ERROR] DotNetNuke.Services.Exceptions.Exceptions - FriendlyMessage="Fehler: Login ist zur Zeit nicht verfügbar." ctrl="DotNetNuke.Web.Mvc.MvcHostControl" exc="System.ServiceModel.Security.SecurityAccessDeniedException: Access is denied.**
> 
> Server stack trace: 
>    at System.ServiceModel.Channels.ServiceChannel.ThrowIfFaultUnderstood(Message reply, MessageFault fault, String action, MessageVersion version, FaultConverter faultConverter)
>    at System.ServiceModel.Channels.ServiceChannel.HandleReply(ProxyOperationRuntime operation, ProxyRpc& rpc)
>    at System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
>    at System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
>    at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

I've already checked all the required iis features, sql settings, .net version, and all the other stuff. Also, I tried to use the same DB 
 in the other server and everything is working there.

I've been totally stuck. 

Could you help me with it?

**Comment by bdukes** [comment]
Does your `web.config` file have `<trust level="Medium" />` in it? Medium Trust is considered obsolete and DNN requires the trust level to be `Full` (or to remove/comment the `<trust/>` element all together)

**Comment by MichaelMatsiy** [comment, marked as answer]
So, the problem was in the endpoint :)
